op-chain-ops/script/script.go

```diff
@@ -361,6 +391,19 @@ func (h *Host) unwindCallstack(depth int) {
 		if len(h.callStack) > 1 {
 			parentCallFrame := h.callStack[len(h.callStack)-2]
 			if parentCallFrame.Prank != nil {
+				if parentCallFrame.Prank.Broadcast {
+					currentFrame := h.callStack[len(h.callStack)-1]
+					bcast := NewBroadcastFromCtx(currentFrame.Ctx)
```

Comment:
Ok, sorry, one last issue; I believe we might be capturing a `broadcast` here that is actually `STATICCALL` (or `view`/`pure` in solidity), which should not be broadcast, as it does not affect the state. When we start the prank we might need to remember the opcode, to correctly tell whether or not the `OnBroadcast` call here should be called.

Reply:
Nice catch - fixed with a check for `STATICCALL`.